src/test/scala/com/microsoft/hyperspace/goldstandard/TPCDSBase.scala

```diff
@@ -0,0 +1,575 @@
+        |`hd_dep_count` INT,
+        |`hd_vehicle_count` INT
+      """.stripMargin,
+    "item" ->
+      """
+        |`i_item_sk` INT,
+        |`i_item_id` CHAR(16),
+        |`i_rec_start_date` DATE,
+        |`i_rec_end_date` DATE,
+        |`i_item_desc` VARCHAR(200),
+        |`i_current_price` DECIMAL(7,2),
+        |`i_wholesale_cost` DECIMAL(7,2),
+        |`i_brand_id` INT,
+        |`i_brand` CHAR(50),
+        |`i_class_id` INT,
+        |`i_class` CHAR(50),
+        |`i_category_id` INT,
+        |`i_category` CHAR(50),
+        |`i_manufact_id` INT,
+        |`i_manufact` CHAR(50),
+        |`i_size` CHAR(20),
+        |`i_formulation` CHAR(20),
+        |`i_color` CHAR(20),
+        |`i_units` CHAR(10),
+        |`i_container` CHAR(10),
+        |`i_manager_id` INT,
+        |`i_product_name` CHAR(50)
+      """.stripMargin,
+    "income_band" ->
+      """
+        |`ib_income_band_sk` INT,
+        |`ib_lower_bound` INT,
+        |`ib_upper_bound` INT
+      """.stripMargin,
+    "promotion" ->
+      """
+        |`p_promo_sk` INT,
+        |`p_promo_id` CHAR(16),
+        |`p_start_date_sk` INT,
+        |`p_end_date_sk` INT,
+        |`p_item_sk` INT,
+        |`p_cost` DECIMAL(15,2),
+        |`p_response_target` INT,
+        |`p_promo_name` CHAR(50),
+        |`p_channel_dmail` CHAR(1),
+        |`p_channel_email` CHAR(1),
+        |`p_channel_catalog` CHAR(1),
+        |`p_channel_tv` CHAR(1),
+        |`p_channel_radio` CHAR(1),
+        |`p_channel_press` CHAR(1),
+        |`p_channel_event` CHAR(1),
+        |`p_channel_demo` CHAR(1),
+        |`p_channel_details` VARCHAR(100),
+        |`p_purpose` CHAR(15),
+        |`p_discount_active` CHAR(1)
+      """.stripMargin,
+    "reason" ->
+      """
+        |`r_reason_sk` INT,
+        |`r_reason_id` CHAR(16),
+        |`r_reason_desc` CHAR(100)
+      """.stripMargin,
+    "ship_mode" ->
+      """
+        |`sm_ship_mode_sk` INT,
+        |`sm_ship_mode_id` CHAR(16),
+        |`sm_type` CHAR(30),
+        |`sm_code` CHAR(10),
+        |`sm_carrier` CHAR(20),
+        |`sm_contract` CHAR(20)
+      """.stripMargin,
+    "time_dim" ->
+      """
+        |`t_time_sk` INT,
+        |`t_time_id` CHAR(16),
+        |`t_time` INT,
+        |`t_hour` INT,
+        |`t_minute` INT,
+        |`t_second` INT,
+        |`t_am_pm` CHAR(2),
+        |`t_shift` CHAR(20),
+        |`t_sub_shift` CHAR(20),
+        |`t_meal_time` CHAR(20)
+      """.stripMargin
+  )
+
+  val tableNames: Iterable[String] = tableColumns.keys
+
+  def createTable(
+      spark: SparkSession,
+      tableName: String,
+      format: String = "parquet",
+      options: Seq[String] = Nil): Unit = {
+    spark.sql(
+      s"""
+         |CREATE TABLE `$tableName` (${tableColumns(tableName)})
+         |USING $format
+         |${options.mkString("\n")}
+       """.stripMargin)
+  }
+
+  private val originalCBCEnabled = conf.cboEnabled
+  private val originalJoinReorderEnabled = conf.joinReorderEnabled
+
```

Comment:
removed `val originalPlanStatsEnabled` from source. It is only required for stats based tests which are not yet supported